packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -117,11 +117,14 @@ Media.createMediaConnection = (
   isMultistream: boolean,
   debugId: string,
   options: {
-    mediaProperties?: {
+    mediaProperties: {
```

Comment:
jsdoc needs updating that it's no longer optional

Reply:
done